### DISCUSSION TITLE
Error notification invalid input sintax for type uuid: "undefined"

### DISCUSSION THREAD

**Discussion by Albertcuicas** [devchallenges-io/curriculum#10] [Report a bug] [answered]
This error is shown when i see https://devchallenges.io/learn or any roadmap being unlogged
![image](https://github.com/user-attachments/assets/1f024e86-d5d3-42b3-b6fc-4f0aca80cb79)

**Comment by nghiemthu** [comment, marked as answer]
@Albertcuicas Thank you for reporting. It should be fixed now 🙏
